packages/devtools/web/index.html

```diff
@@ -49,6 +49,14 @@
     </nav>
 
     <div id="global-actions" class="masthead-nav"></div>
+
+    <div class="masthead-nav">
+        <span horiz-padding class="masthead-item">•</span>
+        <div class="masthead-item action-button active" id="send-feedback-button"
+             title="Send Feedback">
+            <span class="octicon octicon-comment"></span>
+        </div>
+    </div>
```

Comment:
Is there are reason we aren't adding this button the same way we added hot reload and hot restart (in the globalActions container)?

Reply:
Yeah, this is round-about, but the goal here is to minimize or eliminate re-layout of the UI from the straight declarative bits defined in the html file, to the procedural stuff that we build after the compiled javascript has a chance to run.